src/openfermion/utils/_lcu_util.py

```diff
@@ -0,0 +1,181 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Utility methods for LCU circuits."""
+
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
For consistency with our style guide, we'd prefer you import them both in the same line. Also, what is the precise reason you need to import these things?

Reply:
Done.

I imported division because I do divisions in this file, and without this import they would potentially be integer divisions and round down. Actually, I think we should unconditionally include this future import in every file to avoid possible compatibility issues between python 2 and python 3.